spikes/config/main.go

```diff
@@ -37,4 +38,9 @@ func main() {
 	bytesRead, err := configFile.Read(data)
 	fatal(err)
 
+	config := make(map[string]string)
+	err = json.Unmarshal(data[:bytesRead], &config)
+	fatal(err)
```

Comment:
@aThorp96 We set those keys in the config file. I have no idea what kinds of things they might be right now. I suppose stuff like `server_url`, `username`, and `password`? Anyway, you can iterate over the keys in the map from go to get a list of all of the defined ones using a for loop: `for key, _ := range mapThing {}`